pkg/kubelet/kubelet.go

```diff
@@ -108,6 +110,9 @@ func NewMainKubelet(
 		etcdClient:                     etcdClient,
 		kubeClient:                     kubeClient,
 		rootDirectory:                  rootDirectory,
+		tmpfsRootDirectory:             path.Join(rootDirectory, "tmpfs"), // TODO: pass in from cli
```

Comment:
Can you capture this in a function alongside getPluginsDir() and getPodsDir() ?  That way - all of the root-dir schema is roughly in the same area.